inc/mailcollector.class.php

```diff
@@ -1369,14 +1369,16 @@ function getHeaders(\Laminas\Mail\Storage\Message $message) {
       $mail_details = [];
 
       // Construct to and cc arrays
-      $h_tos   = $message->getHeader('to');
       $tos     = [];
-      foreach ($h_tos->getAddressList() as $address) {
-         $mailto = Toolbox::strtolower($address->getEmail());
-         if ($mailto === $this->fields['name']) {
-            $to = $mailto;
+      if (isset($message->to)) {
+         $h_tos   = $message->getHeader('to');
```

Comment:
Since `getHeader` throws an exception, it would be better to catch it I guess. 
Alson, I'm a bit surprised the `to` parameter has public access :)

Reply:
The isset checks header presence, so it should not throw an exception.
```
    public function __isset($name)
    {
        return $this->getHeaders()->has($name);
    }
```